core/src/main/kotlin/ch/ergon/dope/resolvable/operator/FunctionOperator.kt

```diff
@@ -18,6 +19,14 @@ interface FunctionOperator {
         it.queryString
     }
 
+    fun toFunctionQueryString(symbol: String, quantifier: AggregateQuantifier, vararg arguments: DopeQuery) = arguments.joinToString(
```

Comment:
You only need the queryString of the DopeQuery, better change the type of arguments to String and pass the _queryString_ directly

Reply:
done